src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,5 @@
 *REMOVED*Microsoft.CodeAnalysis.Compilation.CreateFunctionPointerTypeSymbol(Microsoft.CodeAnalysis.ITypeSymbol returnType, Microsoft.CodeAnalysis.RefKind returnRefKind, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.ITypeSymbol> parameterTypes, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.RefKind> parameterRefKinds) -> Microsoft.CodeAnalysis.IFunctionPointerTypeSymbol
+Microsoft.CodeAnalysis.CommandLineArguments.GeneratedFilesDirectory.get -> string
```

Comment:
>GeneratedFilesDirectory [](start = 44, length = 23)

nit: there is a small inconsistency in naming between GeneratedFilesOutputPath and GeneratedFilesDirectory. Is that intentional?

Reply:
We have OutputPath and OutputDirectory, so I just made it match the same pattern. No objection to unifying it though.

---
In reply to: [478710831](https://github.com/dotnet/roslyn/pull/47047#discussion_r478710831) [](ancestors = 478710831)